documentation/getting_started/developer_reference/contract_addresses.md

```diff
@@ -0,0 +1,26 @@
+# Deployed DVM Contracts
+Below is where you can find smart contract addresses for UMA-supported mainnet and testnet deployments of the DVM. 
+
+## UMA Token Holders
+If you are a UMA token holder, you will probably only interact with `Voting`, `Finder`, and `DesignatedVotingFactory`. 
```

Comment:
I would add the Governor to this list.

Reply:
Sorry if I gave you conflicting feedback before.

Reply:
Sounds good, I'll promote it to the same line here.

Reply:
Oh, man, I really didn't read that carefully. I'm sorry. Didn't realize you had an entire line about the governor below. Sorry about that :/

Reply:
No problem - don't think it's a big deal!